bundles/org.openhab.binding.wled/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -266,6 +273,52 @@
 		<category>Time</category>
 	</channel-type>
 
+	<channel-type id="sleepMode" advanced="true">
+		<item-type>String</item-type>
+		<label>Sleep Mode</label>
+		<description>Timed Light Mode selects how the light will fade or increase when the sleep timer is turned ON.</description>
+		<state min="0" max="3" step="1" pattern="%.0f" readOnly="false">
```

Comment:
As the type is String, I am not sure that min/max/step make sense here.
Maybe you should switch to a Number type ?

Reply:
Must have been a copy paste error.